microsoft/iot-cardboard-js
# Querying adapter method (and some UI updates)

### Summary of changes 🔍 
Created an adapter for querying in ADT Advanced search component. Linked search results with Advanced search results details list as well.

Querying updates from previous PR comments:
- ~~Review if model name needs to be included with same name properties~~ (not in scope)
- Limit of 1000

UI updates to be included in this PR:
- ~~Value text box has a placeholder~~ (will be replaced next PR)
- Iconography for every option
- Divider spacing improvements

### Testing 🧪
- With Mock data:
  -  Open story for Advanced search
  -  Select Car name property
  -  Select Equals operator
  -  Input: 'Model Y' in value field
  -  See results in details list 
- With Real data:
  -  Go to this branch and run the project
  -  Open local story for Advanced search
  -  Query for your Twins

### Checklist ✔️
